doc/api/fs.md

```diff
@@ -2503,9 +2503,9 @@ changes:
 
 Read data from the file specified by `fd`.
 
-`buffer` is the buffer that the data will be written to.
+`buffer` is the buffer that the data will be read from.
```

Comment:
Maybe something like this works?

```suggestion
`buffer` is the buffer that the data which was read from the fd will be written to.
```

Reply:
On second thought... the way it was before is probably best? It's only the other paragraph that was unclear in my opinion.

Reply:
so the old version is correct? which paragraph wasn't clear @ronag